In [1]:
from sklearn.datasets import load_breast_cancer
from sklearn.model_selection import train_test_split
from sklearn.svm import SVC
from sklearn.preprocessing import StandardScaler
from sklearn.metrics import accuracy_score

In [2]:
X,y = load_breast_cancer(return_X_y=True)

In [3]:
scaler= StandardScaler()
X=scaler.fit_transform(X)

In [40]:
X_train, X_test, y_train, y_test = train_test_split(X, y, test_size=0.2, random_state=42)

model = SVC(kernel='rbf', C=1.5, gamma='scale')
model.fit(X_train, y_train)
predictions = model.predict(X_test)
accuracy = accuracy_score(y_test, predictions)
print(f"Model accuracy: {accuracy * 100:.2f}%")

Model accuracy: 98.25%


rbf kernel,C=1 -> acc = 97.37%
c=.5 -> 96.49%
c=.1 -> 94.74%
c=5 -> 98.25%
C=10 -> acc =97.37%

c=100 -> 93.86%  overfitting 

linear kernel -> acc= 95.61%
